src/db/cass/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM python:3.6
+
+WORKDIR /cass-app
+
+ENV NGINX_RTMP_IP=127.0.0.1
+
+ENV NGINX_RTMP_PORT=19350
+
+ENV CASSANDRA_CLUSTER_IPS="192.168.1.15"
+
+ENV CASSANDRA_CLUSTER_PORT="9043"
+
+ENV CERT_FILE_ADDRESS=192.168.1.15:8008/meowl_nginx.crt
+
+ENV FLASK_APP=/cass-app/app.py
+
+RUN apt-get update
+
+RUN apt-get -y install sudo
```

Comment:
`sudo apt-get -y install` ?

Reply:
What about it

Reply:
Shouldn't it be `sudo apt-get -y install` and not `apt-get -y install sudo` ?

Reply:
Ahh nevermind. you're installing sudo

Reply:
sudo install sudo

Reply:
yes install yes